Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +16,32 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        
         public static int PromptInt(string message)
         {
+            
             // To do: call your implementation. 
-            return 0;
+            return Convert.ToInt32(PromptString(message)); ;
         }
 
         public static string PromptString(string message)
         {
             // To do: call your implementation. 
-            return "";
+            Console.WriteLine(message);
+            return Console.ReadLine();
         }
 
         public static float PromptFloat(string message)
         {
             // To do: call your implementation. 
-            return 0;
+            return Convert.ToSingle(PromptString(message), CultureInfo.InvariantCulture);
         }
 
         public static float CalculateBmi(float weight, float height)
         {
             // To do: call your implementation. 
-            return 0;
+            float bmi = (weight / (height * height));
+            return bmi;
```

Comment:
You shouldn't write duplicated code. Use your own functions - as the TODO comments suggest.
After using your own functions please delete those comments. :)